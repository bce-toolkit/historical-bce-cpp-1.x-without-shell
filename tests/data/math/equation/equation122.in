6 6
1 0 0 -1 0 0
2 1 0 -6 -1 0
6 0 0 0 -3 0
0 3 0 -18 0 0
0 0 1 0 -1 0
0 0 1 -2 0 0
# Co(NO3)2+NH3+KI=CoI2.6NH3+KNO3
