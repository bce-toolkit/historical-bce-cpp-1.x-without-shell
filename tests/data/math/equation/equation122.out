1/2Xa
3Xa
Xa
1/2Xa
Xa
